main.go

```diff
@@ -41,34 +41,37 @@ func main() {
 	if err != nil {
 		log.Fatal("getting current dir:", err)
 	}
+
 	corpusDir := filepath.Join(baseDir, corpusFolderName)
 	if strings.HasSuffix(*compiler, "tinygo") {
 		mustrun(*compiler, "clean")
 	}
 	if err != nil {
-		log.Fatal("calling `%v clean`:", *compiler, err)
+		log.Fatalf("calling `%v clean`: %s", *compiler, err)
 	}
 	os.Mkdir(corpusDir, dirMode) // force directory creation if not exist.
 	_, err = os.ReadDir(corpusDir)
 	if err != nil {
 		log.Fatal("reading corpus directory: ", err)
 	}
 
-	// Commence testing logic.
-	for _, repo := range repos {
-		os.Chdir(corpusDir)
-		cloneOrUpdateRepo(repo.Repo)
+	// Commence testing logic. Start from latest repo additions (end of repos).
```

Comment:
I wonder if adding a `-run` regexp parameter similar to `go test` would make sense here instead of changing the order of the yaml?  That's the other reason I pulled the repo into a yaml file so that you could test with a small number of repos rather than changing the order of entries in the yaml file.